.travis.yml

```diff
@@ -9,10 +9,8 @@ addons:
 node_js:
   - "0.10"
 install:
-  - npm install carto@0.12.1
+  - npm install carto@0.16.0
```

Comment:
You should really use 0.16.3 as the versions before contain some regressions. For linting this does not make a difference but maybe is the wrong signal.